app/src/main/res/values-bn/strings.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright (C) 2021-2024 Sebastian Göls
+  ~
+  ~ This program is free software; you can redistribute it and/or modify
+  ~ it under the terms of the GNU General Public License as published by
+  ~ the Free Software Foundation; version 2 of the License only
+  ~
+  ~ This program is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~ GNU General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU General Public License along
+  ~ with this program; if not, write to the Free Software Foundation, Inc.,
+  ~ 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+  -->
+
+<resources>
```

Comment:
Some strings should not be translated as indicated by the `translatable` attribute being set to `false`. Crowdin already takes care of that.

Remove the corresponding entries from the file.

Reply:
This 5 ?

![image](https://github.com/user-attachments/assets/12ae58ab-ad21-4c2f-8c9d-d8b8a9eb9179)

Reply:
Done